internal/binder/function/funcs_array_test.go

```diff
@@ -557,13 +557,6 @@ func TestArrayCommonFunctions(t *testing.T) {
 			},
 			result: []interface{}{0, 0.4, 1.2},
 		},
-		{
-			name: "array_map",
```

Comment:
Suggest to replace with a similar case instead of just deleting it

Reply:
Done.